test/debugger/test-debugger-stop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var common = require('../common');
+var debug = require('_debugger');
+
+var fork = require('child_process').fork;
+
+var scriptToDebug = common.fixturesDir + '/debug-connect.js';
+
+var nodeProcess;
+
+function doTest() {
```

Comment:
You don't need this function and `run` as well. Write the contents of the function in the top level itself.